include/boost/dll/shared_library.hpp

```diff
@@ -511,6 +511,10 @@ class shared_library
         return base_t::suffix();
     }
 
+    static boost::filesystem::path append_decorations(boost::filesystem::path sl) {
```

Comment:
Remove this function from the public interface for now. It has a very bad name. It looks like the function just concatenates strings, but it actually does some heavy system calls and produces different answers, depending on the user's environment.

Reply:
Maybe we can find another name for this method ? I'd like to have it in the public interface to figure out the computed path before the real "load" occurs.

Reply:
Last commit changed the method name to decorate()